webapp/src/selectors.ts

```diff
@@ -2,15 +2,13 @@ import {getConfig} from 'mattermost-redux/selectors/entities/general';
 
 import {createSelector} from 'reselect';
 
-import manifest from './manifest';
+import {GlobalState} from './types/store';
 
-const emptyArray = [];
+const emptyArray: GithubIssueData[] | UnreadsData[] = [];
 
-const getPluginState = (state) => state['plugins-' + manifest.id] || {};
+export const getPluginState = (state: GlobalState) => state['plugins-github'] || {};
```

Comment:
Does this also work? I think we can assume `state['plugins-github']` is defined and `non-null`

```suggestion
export const getPluginState = (state: GlobalState): PluginState => state['plugins-github'];
```